README.md

```diff
@@ -58,6 +58,7 @@ $ kubectl get opsrc upstream-community-operators -o=custom-columns=NAME:.metadat
 NAME                           PACKAGES
 upstream-community-operators   federationv2,svcat,metering,etcd,prometheus,automationbroker,templateservicebroker,cluster-logging,jaeger,descheduler
 ```
+**_Note_**: Please do not install [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml) and [community-operators](deploy/examples/community.operatorsource.cr.yaml) `OperatorSources` on the same cluster. The rule of thumb is to install [community-operators](deploy/examples/community.operatorsource.cr.yaml) on OpenShift clusters and [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml) on plain vanilla Kubernetes clusters.
```

Comment:
nit:
```suggestion
**_Note_**: Please do not install [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml) and [community-operators](deploy/examples/community.operatorsource.cr.yaml) `OperatorSources` on the same cluster. The rule of thumb is to install [community-operators](deploy/examples/community.operatorsource.cr.yaml) on OpenShift clusters and [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml) on "plain vanilla" Kubernetes clusters.
```

Reply:
Why do I need to add the quotes?

Reply:
plain vanilla kubernetes implies that it's an actual product when really it's just a colloquial term. I would even go as far as "plain vanilla kubernetes"

Reply:
Calling something plain vanilla does not imply that it is a product. Would `upstream kubernetes` satisfy you?

Reply:
Just making a point to consider for your best judgement - I will be satisfied either way :)

Reply:
Food for thought: I would go even further to say "Plain" and "Vanilla" are the same thing.

Reply:
Addressed.